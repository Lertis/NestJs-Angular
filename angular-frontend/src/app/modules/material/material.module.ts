import { NgModule } from '@angular/core';
import { MatButtonModule } from '@angular/material/button';
import { MatToolbarModule } from '@angular/material/toolbar';
import { MatCardModule } from '@angular/material/card';
import { MatFormFieldModule } from '@angular/material/form-field';
import { MatIconModule } from '@angular/material/icon';
import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
import { MatGridListModule } from '@angular/material/grid-list';
import { MatDialogModule } from '@angular/material/dialog';
import { MatInputModule } from '@angular/material/input';
import { MatSidenavModule } from '@angular/material/sidenav';
import { MatListModule } from '@angular/material/list';
import { MatTabsModule } from '@angular/material/tabs';

const materialModules = [
	MatButtonModule,
	MatToolbarModule,
	MatCardModule,
	MatFormFieldModule,
	MatIconModule,
	MatProgressSpinnerModule,
	MatGridListModule,
	MatDialogModule,
	MatInputModule,
	MatSidenavModule,
	MatListModule,
	MatTabsModule,
];

@NgModule({
	imports: [...materialModules],
	exports: [...materialModules],
})
export class MaterialModule {}
