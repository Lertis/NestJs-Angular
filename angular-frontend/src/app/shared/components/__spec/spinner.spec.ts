import { ComponentFixture, TestBed, async } from '@angular/core/testing';
import { SpinnerComponent } from '../spinner/spinner.component';
import { MaterialModule } from '../../../modules/material/material.module';

describe('[Component]: SpinnerComponent', () => {
	let component: SpinnerComponent;
	let fixture: ComponentFixture<SpinnerComponent>;

	beforeEach(async(() => {
		TestBed.configureTestingModule({
			imports: [MaterialModule],
			declarations: [SpinnerComponent],
		}).compileComponents();
	}));

	beforeEach(() => {
		fixture = TestBed.createComponent(SpinnerComponent);
		component = fixture.componentInstance;
		fixture.detectChanges();
	});

	it('component should be created', () => {
		expect(component).toBeTruthy();
		expect(component).toMatchSnapshot();
	});
});
