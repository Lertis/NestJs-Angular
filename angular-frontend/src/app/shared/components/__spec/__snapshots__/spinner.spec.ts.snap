// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: SpinnerComponent component should be created 1`] = `SpinnerComponent {}`;
